src/Node/Expression/CallExpression.php

```diff
@@ -48,17 +45,11 @@ protected function compileCallable(Compiler $compiler)
 
                 $compiler->raw(sprintf('->%s', $callable[1]));
             } else {
-                $closingParenthesis = true;
-                $isArray = true;
-                $compiler->raw(sprintf('call_user_func_array($this->env->get%s(\'%s\')->getCallable(), ', ucfirst($this->getAttribute('type')), $this->getAttribute('name')));
+                $compiler->raw(sprintf('$this->env->get%s(\'%s\')->getCallable()', ucfirst($this->getAttribute('type')), $this->getAttribute('name')));
```

Comment:
PHP 7.1 means we can remove using `call_user_func_array()` and use an inline call instead.